src/Components/Web.JS/src/Virtualize.ts

```diff
@@ -20,8 +20,16 @@ function findClosestScrollContainer(element: Element | null): Element | null {
 }
 
 function init(dotNetHelper: any, spacerBefore: HTMLElement, spacerAfter: HTMLElement, rootMargin = 50): void {
+  // Overflow anchoring can cause an ongoing scroll loop, because when we resize the spacers, the browser
+  // would update the scroll position to compensate. Then the spacer would remain visible and we'd keep on
+  // trying to resize it.
+  const scrollContainer = findClosestScrollContainer(spacerBefore);
+  if (scrollContainer) {
+    scrollContainer.style.overflowAnchor = 'none';
```

Comment:
Have we tried this in combination with `scroll-snap-align` and similar things?

Reply:
I've tried it in some basic cases, and whether or not it works well depends on exactly how big the snappable items are and whether you're doing mouse or touch scrolling. In a lot of cases it's quite  Sometimes on desktop, I find that `scroll-snap-align` feels pretty weird even natively (without virtualization).

As a point of comparison, I tried out react-window in conjunction with `scroll-snap-align` (by visiting https://react-window.now.sh/#/examples/list/fixed-size and using the dev tools to apply the extra CSS styling). It behaves somewhat OK, probably better than our `<Virtualize>` in this scenario, but there are cases where react-window glitches out badly with this. If you scroll quickly and happen to release the mouse right in the middle of an element, instead of snapping to the place you let go of the mouse, it scrolls backwards a  long way and snaps to some other element instead.

If we think this is going to be a primary scenario it does seem we're not there yet. However it seems unlikely to me that it would be a core scenario.

Reply:
> If we think this is going to be a primary scenario it does seem we're not there yet. However it seems unlikely to me that it would be a core scenario.

I'm just trying to make sure we cover our backs and think about these types of things, scroll-snap and friends are new right now, but they will become more prevalent in the futre. My goal is that we understand whether or not we think these things will require a big redesign in the future or just improvements to the existing implementation.